website/docs/r/powerbidedicated_capacity.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "PowerBIDedicated"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_powerbidedicated_capacity"
+sidebar_current: "docs-azurerm-resource-powerbidedicated-capacity"
+description: |-
+  Manages a PowerBIDedicated Capacity.
+---
+
+# azurerm_powerbidedicated_capacity
+
+Manages a PowerBIDedicated Capacity.
```

Comment:
It seems that I cannot rename this description as what you suggested since this resource is to create capacity which is the feature of the power bi dedicated.
See more details from here:
Tutorial: https://docs.microsoft.com/en-us/azure/power-bi-embedded/
Rest api: https://docs.microsoft.com/en-us/rest/api/power-bi-embedded/capacities/create
azure-rest-api-spec: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/stable/2017-10-01/powerbidedicated.json